test/unittest/Database/DatabaseDumpTests.cpp

```diff
@@ -72,31 +64,7 @@ constexpr const int16_t MAGNITUDE_DEFAULT = 0;
 #define DATAREADER_DEFAULT_ID(x) EntityId(x * 9 + 8)
 
 #define TIME_DEFAULT(x) std::chrono::system_clock::time_point(std::chrono::steady_clock::duration(x))
-#define GUID_DEFAULT(x) "01.0f.00.00.00.00.00.00.00.00.00.0" + std::to_string(x) + "|00.00.00.00"
-
-
-DatabaseDump load_file(
-        std::string filename)
-{
-    // Check if the file exists
-    std::ifstream file(filename);
-    if (!file.good())
-    {
-        throw BadParameter("File " + filename + " does not exist");
-    }
-
-    // Get the json file
-    DatabaseDump dump;
-    file >> dump;
-
-    // Erase the description tag if existing
-    if (dump.contains(DESCRIPTION_TAG))
-    {
-        dump.erase(DESCRIPTION_TAG);
-    }
-
-    return dump;
-}
+#define GUID_DEFAULT(x) "01.0f.00.00.00.00.00.00.00.00.00.0" + std::to_string(x) + "|0.0.0.0"
```

Comment:
The suggested changed is wrong, it will broke the dump tests. The std::to_string(x) will add the second digit, if we put another zero, there will be the digits.

Reply:
so true!! I misread that :+1: